modules/doc/content/application_development/build_system.md

```diff
@@ -2,9 +2,12 @@
 
 ## Overview
 
-The MOOSE build system is a hierarchical Makefile based system.  The idea is that no matter where you
-are in the hierarchy of MOOSE applications/modules/framework, simply typing `make` will rebuild
-everything that needs to be rebuilt.  The basic way to build any MOOSE application is simply to do a:
+!style! halign=left
+The MOOSE build system is a hierarchical Makefile based system. The idea is that no matter where
+you are in the hierarchy of MOOSE applications/modules/framework, simply typing `make` will rebuild
+everything that needs to be rebuilt. The basic way to build any MOOSE application is simply to do
+a:
+!style-end!
```

Comment:
FYI - if you wanted to reduce new line count, simply placing `!style halign=left` in front of the paragraph and not the full "start-to-end" syntax is sufficient to get the spacing right after a heading. Mentioning it in case you wanted to adjust as you're working through other parts of this review, but I will not require it, as this is good syntax as well.

Reply:
Oh! I didn't know that. Yes I'd much rather not need the style-end syntax.